pull_config.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+#
+# Pull config from the production database to the local database.
+#
+
+# Prompt the user before running
+echo
+echo "*******************"
+echo "**    WARNING    **"
+echo "*******************"
+echo
+
+read -p "You're about to update your local config database with the one on production. You sure about this? [y/n] " -r
```

Comment:
Should we specify here that existing local documents sharing the same `_id` value with production documents will **not** be overwritten? 

>If you restore to an existing database, mongorestore will only insert into the existing database, and does not perform >updates of any kind. If existing documents have the same value _id field in the target database and collection, >mongorestore will not overwrite those documents.

http://docs.mongodb.org/manual/reference/program/mongorestore/